tools/serverpod_cli/lib/src/generator/open_api/open_api_objects.dart

```diff
@@ -0,0 +1,1132 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
+
+part of openapi_definition;
```

Comment:
Question: Just out of curiosity, are there any private methods required from openapi_defenition?

Also, effective dart encourages the use of the path instead of the library name, so if this is required please provide the path.
source: https://dart.dev/effective-dart/usage#do-use-strings-in-part-of-directives.